src/state/models/discovery/feeds.ts

```diff
@@ -89,9 +113,24 @@ export class FeedsDiscoveryModel {
   // =
 
   _replaceAll(res: AppBskyUnspeccedGetPopularFeedGenerators.Response) {
+    // 1. set feeds data to empty array
     this.feeds = []
+
+    // 2. push data into feeds array
+    for (const f of res.data.feeds) {
+      this.feeds.push(new CustomFeedModel(this.rootStore, f))
+    }
+
+    // 3. set loadMoreCursor
+    this.loadMoreCursor = res.data.cursor
```

Comment:
Nit: could dry this up by resetting this.feeds = [] and then calling this._append(res), but fine with me if you wanna keep things separate and simple too 👍

Reply:
Yeah true. That is the convention in other model files so I can follow it.